Gemfile

```diff
@@ -8,6 +8,11 @@ gem "rails", "~> 7.0.5"
 
 gem "sprockets-rails" # Asset pipeline
 gem "pg", "~> 1.1" # Postgres database
```

Comment:
Do we need Postgres anymore?

Reply:
Good catch! Updated in the latest commit.